etl/dags/example_etl_galaxies.py

```diff
@@ -63,6 +63,10 @@
             description="Set how close galaxies need ot be to the milkyway in order to be loaded to DuckDB.",
         )
     },
+    # Warning - DuckDB is not a persistent database between workers. To move this workflow in production,
+    # use a persistent DB and remove the max_active_runs and concurrency parameters below
```

Comment:
```suggestion
```
I am not entirely sure how motherduck handles concurrency

Reply:
They might not allow it yet, but I think this serves our purpose!